demos/distance-field-3.c

```diff
@@ -305,16 +305,9 @@ void display( GLFWwindow* window )
     glEnable( GL_BLEND );
     glBlendFunc( GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA );
 
-    texture_glyph_t * glyph = texture_font_get_glyph( font, "@");
-
     int width, height;
     glfwGetFramebufferSize( window, &width, &height );
 
-    float glyph_height = glyph->height * width/(float)glyph->width;
-    float glyph_width  = glyph->width * height/(float)glyph->height;
-    int x = -glyph_width/2 + width/2.;
-    int y = -glyph_height/2 + height/2.;
```

Comment:
Even if not used, this is relevant information. Maybe commenting those line would be better, no ?